deno/payloads/v10/guild.ts

```diff
@@ -117,7 +117,7 @@ export interface APIGuild extends APIPartialGuild {
 	 */
 	afk_channel_id: Snowflake | null;
 	/**
-	 * afk timeout in seconds
+	 * afk timeout in seconds, can be set to: `60`, `300`, `900`, `1800`, `3600`
 	 */
 	afk_timeout: number;
```

Comment:
Is there a reason why the type wasn't changed to `60 | 300 | 900 | 1800 | 3600`?

Reply:
Forgotten is the reason

Reply:
Sorry, I forgot to change that, now was added in #590 👍